bus/bus.go

```diff
@@ -435,7 +440,7 @@ func (b *Bus) Handler() http.Handler {
 		"PUT    /hosts/blocklist":                b.hostsBlocklistHandlerPUT,
 		"POST   /hosts/pricetables":              b.hostsPricetableHandlerPOST,
 		"POST   /hosts/remove":                   b.hostsRemoveHandlerPOST,
-		"POST   /hosts/scans":                    b.hostsScanHandlerPOST,
+		"POST   /hosts/scan":                     b.hostsScanHandlerPOST,
```

Comment:
feels like this could/should be `POST /host/:hostkey/scan`

Looking at the request type:
```go
HostScanRequest struct {
		HostKey types.PublicKey `json:"hostKey"`
		HostIP  string          `json:"hostIP"`
		Timeout DurationMS      `json:"timeout"`
}
```
it always felt like the IP should be implied by the `hk`, the key should be part of the route and the timeout should be enforced by the request context